### PR TITLE
[docs] Warning about Istio and admission-policy-engine restrictions

### PR DESCRIPTION
**Pull request by kirill-kashin**
## Description

When using Deckhouse version >= 1.55, the admission-policy-engine module works in `Baseline` mode by default. In this regard, Istio's init-containers can't configure rules for application's traffic interception, so the Pod's can't start. The solution added to docs.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: fix
summary: Istio and admission-policy-engine restrictions clarifications.
```

### CODE REVIEW

File: modules/015-admission-policy-engine/docs/README_RU.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Pod Security Standards определяют три политики, охват
 
 {% alert level="info" %}
 Модуль не применяет политики к системным пространствам имен.
+В версии Deckhouse >= 1.55. Контейнер `sidecar-injector` Istio запускается только в режиме `Privileged`.
```
Comment (z9r5) [inline modules/015-admission-policy-engine/docs/README_RU.md:12]:
```suggestion
При использовании Deckhouse версии >= 1.55, контейнер `sidecar-injector` Istio запускается только в режиме `Privileged`.
```

File: modules/015-admission-policy-engine/docs/README.md [hunk 2]
```diff
@@ -9,7 +9,8 @@ The Pod Security Standards define three different policies to broadly cover the
 
 {% alert level="info" %}
 The module does not apply policies to system namespaces.
-{% endalert %}    
+In deckhouse version >= 1.55. Istio sidecar-injector container starts only in `Privileged` mode
```
Comment (z9r5) [inline modules/015-admission-policy-engine/docs/README.md:12]:
```suggestion
When using Deckhouse version >= 1.55, the Istio `sidecar-injector` container runs only in `Privileged` mode.
```

**Review by z9r5 — APPROVED** [review]

File: modules/110-istio/docs/README_RU.md [hunk 3]
```diff
@@ -179,6 +179,12 @@ Istio позволяет осуществлять сбор трейсов с п
 Основная цель активации — добавить sidecar-контейнер к подам приложения, после чего Istio сможет управлять трафиком.
 
 Рекомендованный способ добавления sidecar-ов — использовать sidecar-injector. Istio умеет «подселять» к вашим подам sidecar-контейнер с помощью механизма [Admission Webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/). Настраивается с помощью лейблов и аннотаций:
+
+{% alert level="info" %}
+Версия deckhouse >= 1.55 При работе модуля admission-policy-engine Istio sidecar-injector контейнеры работаю только в режиме `Privileged`
+[admission-policy-engine](https://deckhouse.ru/documentation/v1/modules/015-admission-policy-engine/)
```
Comment (z9r5) [inline modules/110-istio/docs/README_RU.md:185]:
```suggestion
При использовании Deckhouse версии >= 1.55 и включенном модуле [admission-policy-engine](../015-admission-policy-engine/), контейнер `sidecar-injector` Istio запускается только в режиме `Privileged`.
```

File: modules/110-istio/docs/README.md [hunk 4]
```diff
@@ -175,6 +175,11 @@ The istiod controller and sidecar-proxy containers export their own metrics that
 The main purpose of the activation is to add a sidecar container to the application Pods so that Istio can manage the traffic.
 
 The sidecar-injector is a recommended way to add sidecars. Istio can inject sidecar containers into user Pods using the [Admission Webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) mechanism. You can configure it using labels and annotations:
+{% alert level="info" %}
+Deckhouse version >= 1.55 When admission-policy-engine Istio sidecar-injector module is running, containers work only in `Privileged` mode
+[admission-policy-engine](https://deckhouse.io/documentation/v1/modules/015-admission-policy-engine/)
```
Comment (z9r5) [inline modules/110-istio/docs/README.md:180]:
```suggestion
When using Deckhouse version >= 1.55 and the [admission-policy-engine] module is enabled(../015-admission-policy-engine/), the `sidecar-injector` Istio container runs only in `Privileged` mode.
```

File: modules/110-istio/docs/README.md [hunk 5]
```diff
@@ -175,6 +175,10 @@ The istiod controller and sidecar-proxy containers export their own metrics that
 The main purpose of the activation is to add a sidecar container to the application Pods so that Istio can manage the traffic.
 
 The sidecar-injector is a recommended way to add sidecars. Istio can inject sidecar containers into user Pods using the [Admission Webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) mechanism. You can configure it using labels and annotations:
+{% alert level="info" %}
+When using Deckhouse version >= 1.55 and the [admission-policy-engine] module is enabled(../015-admission-policy-engine/), the `sidecar-injector` Istio container runs only in `Privileged` mode.
```
Comment (z9r5) [inline modules/110-istio/docs/README.md:179]:
```suggestion
When using Deckhouse version >= 1.55 and the [admission-policy-engine](../015-admission-policy-engine/) module is enabled, the `sidecar-injector` Istio container runs only in `Privileged` mode.
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: modules/015-admission-policy-engine/docs/README_RU.md [hunk 6]
```diff
@@ -9,6 +9,7 @@ Pod Security Standards определяют три политики, охват
 
 {% alert level="info" %}
 Модуль не применяет политики к системным пространствам имен.
+При использовании Deckhouse версии >= 1.55, контейнер `sidecar-injector` Istio запускается только в режиме `Privileged`.
```
Comment (z9r5) [inline modules/015-admission-policy-engine/docs/README_RU.md:12]:
```suggestion

При использовании Deckhouse версии >= 1.55, контейнер `sidecar-injector` Istio запускается только в режиме `Privileged`.
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: modules/015-admission-policy-engine/docs/README.md [hunk 7]
```diff
@@ -9,7 +9,9 @@ The Pod Security Standards define three different policies to broadly cover the
 
 {% alert level="info" %}
 The module does not apply policies to system namespaces.
-{% endalert %}    
+
+When using Deckhouse version >= 1.55, the Istio `sidecar-injector` container runs only in `Privileged` mode.
```
Comment (apolovov) [inline modules/015-admission-policy-engine/docs/README.md:13]:
```suggestion
When using Deckhouse version >= 1.55, the module works in `Baseline` mode by default. In this regard, Istio's init-containers can't configure rules for application's traffic interception, so the Pod's can't start. The solution is either configure the module to work in `Restricted` mode or add a label `security.deckhouse.io/pod-policy=privileged` to application's Namespace.
```

File: modules/110-istio/docs/README.md [hunk 8]
```diff
@@ -175,6 +175,10 @@ The istiod controller and sidecar-proxy containers export their own metrics that
 The main purpose of the activation is to add a sidecar container to the application Pods so that Istio can manage the traffic.
 
 The sidecar-injector is a recommended way to add sidecars. Istio can inject sidecar containers into user Pods using the [Admission Webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) mechanism. You can configure it using labels and annotations:
+{% alert level="info" %}
+When using Deckhouse version >= 1.55 and the [admission-policy-engine](../015-admission-policy-engine/) module is enabled, the `sidecar-injector` Istio container runs only in `Privileged` mode.
```
Comment (apolovov) [inline modules/110-istio/docs/README.md:179]:
```suggestion
When using Deckhouse version >= 1.55 and the [admission-policy-engine](../015-admission-policy-engine/) module is enabled, it works in `Baseline` mode by default. In this regard, Istio's init-containers can't configure rules for application's traffic interception, so the Pod's can't start. The solution is either configure the admission-policy-engine module to work in `Restricted` mode or add a label `security.deckhouse.io/pod-policy=privileged` to application's Namespace.
```

File: modules/110-istio/docs/README_RU.md [hunk 9]
```diff
@@ -179,6 +179,11 @@ Istio позволяет осуществлять сбор трейсов с п
 Основная цель активации — добавить sidecar-контейнер к подам приложения, после чего Istio сможет управлять трафиком.
 
 Рекомендованный способ добавления sidecar-ов — использовать sidecar-injector. Istio умеет «подселять» к вашим подам sidecar-контейнер с помощью механизма [Admission Webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/). Настраивается с помощью лейблов и аннотаций:
+
+{% alert level="info" %}
+При использовании Deckhouse версии >= 1.55 и включенном модуле [admission-policy-engine](../015-admission-policy-engine/), контейнер `sidecar-injector` Istio запускается только в режиме `Privileged`.
```
Comment (apolovov) [inline modules/110-istio/docs/README_RU.md:184]:
```suggestion
В Deckhouse версии >= 1.55 модуль [admission-policy-engine](../015-admission-policy-engine/) по умолчанию работает в режиме `Baseline`. Если его включить, init-контейнеры Istio не могут добавить необходимые правила для перехвата прикладного трафика и Поды не могут запуститься. Решение — либо поменять политику по умолчанию на `Restricted`, либо выставить эту политику для отдельного приложения с помощью лейбла `security.deckhouse.io/pod-policy=privileged` на прикладной Namespace.
```

**Review by apolovov — CHANGES_REQUESTED** [review]

**Review by apolovov — APPROVED** [review]

**Pull request merged**